pkg/querier/worker/util_test.go

```diff
@@ -22,14 +22,14 @@ func (h HandlerFunc) Do(ctx context.Context, req queryrangebase.Request) (queryr
 
 func TestHandleQueryRequest(t *testing.T) {
 	ctx := user.InjectOrgID(context.Background(), "1")
-	request, err := queryrange.QueryRequestWrap(ctx, &queryrange.LokiRequest{})
+	request, err := queryrange.DefaultCodec.QueryRequestWrap(ctx, &queryrange.LokiRequest{})
```

Comment:
Should we not iterate over all supported codecs to validate that they all work?

Reply:
So far we have only this one. Enterprise has another. Codec might be a misleading name.